apps/fortuna/src/command/run.rs

```diff
@@ -25,23 +29,40 @@ use {
         Result,
     },
     axum::Router,
+    ethers::{
+        middleware::Middleware,
+        providers::{
+            Http,
+            Provider,
+        },
+        signers::{
+            LocalWallet,
+            Signer,
+        },
+        types::Address,
+    },
     std::{
         collections::HashMap,
         net::SocketAddr,
         sync::Arc,
+        time::Duration,
     },
     tokio::{
         spawn,
         sync::watch,
+        time,
     },
     tower_http::cors::CorsLayer,
     utoipa::OpenApi,
     utoipa_swagger_ui::SwaggerUi,
 };
 
+const TRACK_DURATION: Duration = Duration::from_secs(10);
```

Comment:
Although the type is `Duration` but I think `TRACK_INTERVAL` is a better name for this